lib/linguist/languages.yml

```diff
@@ -4701,6 +4693,26 @@ SPARQL:
   - ".sparql"
   - ".rq"
   language_id: 331
+Starlark:
+  type: programming
+  tm_scope: source.python
+  ace_mode: python
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
+  color: "#43a047"
```

Comment:
Since Starlark is [a part of the Bazel family](https://github.com/bazelbuild/starlark) I’ve defined the color as a [Bazel green one](https://blog.bazel.build/2017/07/05/new-logo-and-homepage.html) ([preview](https://www.colorhexa.com/43a047)).